src/db/store-db.ts

```diff
@@ -28,7 +28,9 @@ const conflictsDesign: JsDesignDocument = {
     conflictRevs: {
       map: stringifyCode(function (doc) {
         if (doc._conflicts != null) {
-          emit()
+          // Use the value property on the view document to contain all the
```

Comment:
Can you put the comment outside the method, so we don't upload it to the server?